cinema/views.py

```diff
@@ -0,0 +1,38 @@
+from rest_framework.decorators import api_view
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+from rest_framework import status
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    else:
+        serializer = MovieSerializer(data=request.data)
+        if serializer.is_valid():
+            serializer.save()
```

Comment:
In the `movie_list` view, when creating a new movie, the `MovieSerializer` should be initialized with `many=False` since it handles a single instance creation. However, this is not explicitly required as the default behavior is for a single instance.